app/javascript/selectors/answersSelector.js

```diff
@@ -32,6 +33,28 @@ export function getNumUnanswered(questions, buildingId, state) {
   }, 0);
 }
 
+export function getNumAnswered(questions, buildingId, state) {
+  return questions.reduce((count, question) => {
+    if (isUnansweredQuestion(question, buildingId, state)) {
+      return count;
+    }
+    return count + 1;
+  }, 0);
+}
+
+export function getAllActiveAnswersForCategory(categoryId, buildingId, state) {
```

Comment:
TODO: change this to the fixed selector methods in `kevin/display-all-qs` once it's merged in

(no changes needed atm, adding this for future reference)